scripts/generateDockerfile

```diff
@@ -17,9 +17,25 @@ RUN yarn --version
 
 RUN apt update
 
+# for downloading shfmt
+RUN $APT_INSTALL $SHFMT_BUILD_DEPS && apt-mark auto $SHFMT_BUILD_DEPS
+
+# shfmt is needed for shell command parsing
+RUN wget https://github.com/mvdan/sh/releases/download/v3.5.0/shfmt_v3.5.0_linux_amd64 -O /bin/shfmt && \
+  echo "8feea043364a725dfb69665432aee9e85b84c7f801a70668650e8b15452f6574  /bin/shfmt" | sha256sum --check && \
+  chmod +x /bin/shfmt && \
+  shfmt --version
+
+
+# Needed for building RocksDB bindings for Node.js
+RUN $APT_INSTALL $ROCKSDB_BUILD_DEPS && apt-mark auto $ROCKSDB_BUILD_DEPS
+
 # Git is needed for both pre-commit and also for cloning repositories before
 # running the bot's commands
 RUN $APT_INSTALL git && git --version
+
+RUN git config --global user.name command-bot && \
+  git config --global user.email "opstooling+commandbot@parity.io"
```

Comment:
Please combine all commands into one RUN. It will minimize number of image layers (https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#minimize-the-number-of-layers)

Reply:
Let's skip it for now, I'll fix it during ci refactor

Reply:
@alvicsam if you're planning on touching that, I was thingking maybe we could go away from generating dockerfiles into having them in plain `Dockerfile` files?  
It's usually much easier to navigate through projects this way.  

WDYT?

Reply:
Yes, I also had the same intention. I don't see the reason why it should be generated such way

Reply:
im not fan of generating it, but as I saw it's handy in terms of reusing same set of dependencies for CI and App builds.. but if same could be done without copy/paster - that would be interesting to see